android-lib/src/androidTest/java/com/kinvey/androidTest/network/MockMultiInsertNetworkManager.java

```diff
@@ -36,14 +37,20 @@ public Save saveBlocking(T entity) throws IOException {
     @Override
     public SaveBatch saveBatchBlocking(List<T> list) throws IOException {
         useMultiInsertSave = true;
+        multiPostCount++;
         return super.saveBatchBlocking(list);
     }
 
     public void clear() {
+        multiPostCount = 0;
```

Comment:
also wondering if this method needs to be thread-safe

Reply:
Created as atomic variable to make sure that is thread-safe.